maquetado/scss/styles.scss

```diff
@@ -107,3 +113,25 @@
 .height {
   line-height: 24px;
 }
+
+.navbar {
+  background-color: $white;
+  box-shadow: 0 4px 10px 0 $black-05;
+  height: 70px;
+  position: sticky;
```

Comment:
This property is for fixing an element based on the user's scroll position. In this case it's easier to set `position: fixed` if you want it to be on top of the screen since it's already going to be there if you put the nav tag right after the body tag.